src/mds/CInode.cc

```diff
@@ -1037,12 +1039,12 @@ void CInode::fetch(MDSInternalContextBase *fin)
   object_t oid = CInode::get_object_name(ino(), frag_t(), "");
   object_locator_t oloc(mdcache->mds->mdsmap->get_metadata_pool());
 
+  // Old on-disk format: inode stored in xattr of a dirfrag
```

Comment:
Hmm, did you look at maybe just getting rid of that? The old format is 4.5 years old at this point and we can probably put something in to block it politely.